contracts/src/CrossChainApplications/NativeTokenBridge/INativeTokenDestination.sol

```diff
@@ -0,0 +1,57 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: Ecosystem
+
+pragma solidity 0.8.18;
+
+import {TeleporterFeeInfo} from "../../Teleporter/ITeleporterMessenger.sol";
+
+/**
+ * @dev Interface that describes functionalities for a contract that can mint native tokens when
+ * paired with a {INativeTokenSource} or {IERC20TokenSource} contract that will lock tokens on another chain.
+ */
+interface INativeTokenDestination {
+    /**
+     * @dev Emitted when tokens are burned in the destination contract and to be unlocked on the source contract.
+     */
+    event TransferToSource(
+        address indexed sender,
+        address indexed recipient,
+        uint256 amount,
+        uint256 teleporterMessageID
```

Comment:
wondering if we'd want to index `teleporterMessageID` similar fo the source contracts.

Reply:
think they can be useful to index alongside the existing `ITeleporterMessenger` events to get an easy tracking of the flow of a Teleporter message ID